hw/ip/aes/doc/dv/index.md

```diff
@@ -60,22 +64,61 @@ It can be created manually by invoking [`regtool`]({{< relref "util/reggen/READM
 All test sequences reside in `hw/ip/aes/dv/env/seq_lib`.
 The `aes_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves as a starting point.
 All test sequences are extended from `aes_base_vseq`.
-`aes_base_vseq` provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
-Some of the most commonly used tasks / functions are as follows:
-* aes_init:    Initialize the AES module from the randomized environment variables in the config.
-* set_op:      Set AES operation to encrypt or decrypt.
-* write_key:   Write initial key to AES init key registers.
-* add_data:    Add the next 128 block to the input registers.
-* read_output:  Poll the status register for data ready bit and read the result from AES output registers.
-* clear_reg:  Based on the input this function clears data input-, data output- or key-registers or any combination of these.
-* set_manual_trigger: Chooses between AES auto start and manual start.
-* trigger_start:  Set the start bit to trigger a new encryption/decryption.
+`aes_base_vseq` provides commonly used handles, variables, functions and tasks that the test sequences can simply use / call.
+The tasks can be split into two groups.
+Those that provide basic low level functionality.
+* **aes_init**:    Initialize the AES module from the randomized environment variables in the config.
+* **set_op**:      Set AES operation to encrypt or decrypt.
+* **write_key**:   Write initial key to AES init key registers.
+* **add_data**:    Add the next 128 block to the input registers.
+* **read_output**:  Poll the status register for data ready bit and read the result from AES output registers.
+* **clear_reg**:  Based on the input this function clears data input-, data output- or key-registers or any combination of these.
+* **set_manual_trigger**: Chooses between AES auto start and manual start.
+* **trigger_start**:  Set the start bit to trigger a new encryption/decryption.
+And those that provide more complex functionality.
```

Comment:
I think the text would be clearer structured like this:
```
The tasks can be split into two groups: those that provide basic low level functionality and those that provide more complex functionality.
Simple tasks include:
* **task1**: blah blah
* ...
More complex tasks include:
* **task10*: blah blah
```

Reply:
thanks